examples/snippets/package.json

```diff
@@ -11,13 +11,8 @@
     "test": "echo 'Not Applicable'",
     "test:e2e": "echo 'Not Applicable'",
     "build": "echo 'Not Applicable'",
-    "lint": "eslint --ext .ts,.js --no-error-on-unmatched-pattern .",
-    "lint-fix": "eslint --fix --ext .ts,.js --no-error-on-unmatched-pattern ."
-  },
-  "lint-staged": {
-    "*.{js,ts}": "npm run lint-fix",
-    "*.json": "prettier --write --single-quote false",
-    "*.yaml": "prettier --write"
+    "lint": "biome lint .",
```

Comment:
We can remove the whole `lint-staged` block since it's inherited from the top-level `package.json` at the root of the project.